core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -149,6 +152,59 @@ boolean testClassLoad(final String className, final String asTypeName)
    */
   void waitForBalance() throws AccumuloException;
 
+  /**
+   * Throws an exception if a tablet server can not be contacted.
+   *
+   * @param args
+   *          Command line arguments passed in from user command.
+   * @param secretOption
+   *          Specified instance secret to use for commands.
+   *
+   * @since 2.1.0
+   */
+  boolean fateFail(List<String> args, String secretOption) throws AccumuloException;
```

Comment:
Wondering if it would be better to use char[] or a byte array instead of a string?  Even better would be if the hashed value was used.  If this is mirroring other places in the code, then maybe as a follow on issue as a general improvement in password best practices.